Documentation/ApiOverview/CropVariants/General/Index.rst

```diff
@@ -0,0 +1,139 @@
+.. include:: ../../Includes.txt
+
+.. _cropvariants:
+
+=====================
+General Configuration
+=====================
+
+The following examples are basically meant to add one single
+cropping configuration to sys_file_reference,  which will then apply in every
+record, which reference images.
+
+In this example we configure two crop variants, one with the id "mobile",
+one with the id "desktop". The array key defines the crop variant id, which will be used
+when rendering an image with the image view helper.
```

Comment:
how will it be used in the image view helper?

Reply:
Please have a look at the end of the page:
`<f:image image="{data.image}" cropVariant="mobile" width="800" />`